kobo-bookmark-extractor.py

```diff
@@ -76,8 +77,12 @@ def handle_data(self, data):
             output_descriptor = open(os.path.basename(book_file[20:-5]) + '.html', 'w')
             output_descriptor.write('<meta charset="UTF-8"><style>body { font-family: sans-serif; }</style>')
 
-        chapter_file, start_container_path_point = start_container_path.split('#', 1)
-        _, end_container_path_point = end_container_path.split('#', 1)
+        try:
+            chapter_file, start_container_path_point = start_container_path.split('#', 1)
+            _, end_container_path_point = end_container_path.split('#', 1)
+        except:
```

Comment:
When does this happen? What are `start_container_path` and `end_container_path` set to in those cases? I'm wondering if we add support for those cases :)